CHANGELOG.md

```diff
@@ -14,16 +20,24 @@
   [[GH-236]](https://github.com/digitalocean/csi-digitalocean/pull/236)
 * Support checkLimit for multiple pages
   [[GH-235]](https://github.com/digitalocean/csi-digitalocean/pull/235)
+* Log snapshot responses
+  [[GH-234]](https://github.com/digitalocean/csi-digitalocean/pull/234)
 * Return error when fetching the snapshot fails
   [[GH-233]](https://github.com/digitalocean/csi-digitalocean/pull/233)
-* Reject requests for block access type
-  [[GH-225]](https://github.com/digitalocean/csi-digitalocean/pull/225)
+* Refactor waitAction
+  [[GH-229]](https://github.com/digitalocean/csi-digitalocean/pull/229)
 * Assume detached state on 404 during ControllerUnpublishVolume
   [[GH-221]](https://github.com/digitalocean/csi-digitalocean/pull/221)
-* Implement NodeGetVolumeStats RPC
```

Comment:
Was this change dropped?  Why are we dropping it from the changelog?

Reply:
I changed the order of the change log items to be sorted by Github PR number. If you look on the right-hand side in Github, you should see the item about `NodeGetVolumeStats` being moved to line 35/36.

Reply:
👍